content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 id: intro-sorted-sets
 title: 'More Operations on Sorted Sets'
-author: Darren Yao, Benjamin Qi, Andrew Wang
+author: Darren Yao, Benjamin Qi, Andrew Wang, Aadit Ambadkar
```

Comment:
I don't think this warrants adding yourself as an author. You can add yourself as a contributor though (like in this module: https://github.com/cpinitiative/usaco-guide/blob/f352643e443adea14c1e83eaa07eaf158b3bd87f/content/3_Silver/DFS.mdx)

Reply:
Hm but I did make a lot of changes and additions to the page. If you really insist then I'll make it a contributor

Reply:
doing what an autoformatter would have done doesn't count

Reply:
yeah i agree ^^

listing yourself as a contributor would be more reasonable

Reply:
done!